### PR TITLE
映射文档 add backward etc.

### PR DESCRIPTION
**Pull request by zpceng314**
add backward etc.

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.MultiHeadAttention.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+```
+
+Pytoch 参数更多，具体如下：
+### 参数映射
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
+| embed_dim          | embed_dim            | 表示输入 tensor 的维度。  |
+| num_heads          | num_heads            | 表示 head 的数量。  |
+| dropout          | dropout           | 表示 dropout 概率。  |
+| bias          | bias_attr  | 是否使用偏置， Paddle 支持自定义偏置属性， torch 不支持。  |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.MultiHeadAttention.md]: 加一句 直接将bias赋值给bias_attr即可 吧，这个其实是torch是paddle子集，不要让用户觉得好像不支持转

File: docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.backends.cudnn.is_available.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+PaddlePaddle 目前无对应 API，可使用如下代码组合实现该 API。
+
+###  转写示例
+```python
+# Pytorch 写法
+torch.backends.cudnn.is_available()
+
+# Paddle 写法
+bool(paddle.device.get_cudnn_version())
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.backends.cudnn.is_available.md:18]: 这个转换能成功不，将字符串强转bool

File: docs/guides/model_convert/convert_from_pytorch/api_difference/sparse/torch.sparse.sum.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+输入参数不一致和返回类型不一致，具体如下：
+
+### 参数映射
+
+ |PyTorch |  PaddlePaddle |  备注|
+ |--------|  ------------- | ------|
+ |input | x|         输入的 Tensor，仅参数名不一致。|
+ |dim   |      axis   |输入的第二个 Tensor，仅参数名不一致。|
+ |dtype   |      dtype   |输出数据的类型，参数类型不一致。 Pytorch 为 torch.dtype 类型, Paddle 为 str 类型。 需要转写。|
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/sparse/torch.sparse.sum.md]: 这种细微的差异就视作一致吧

File: docs/guides/model_convert/convert_from_pytorch/api_difference/sparse/torch.sparse.sum.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+ |-  |      keepdim   |是否留减少的维度， Pytorch 无此参数， Paddle 保持默认即可。|
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/sparse/torch.sparse.sum.md:24]: 这里关键是返回不同，可以加一行 **返回值**

File: docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.backends.cudnn.is_available.md (hunk 2 above)
Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.backends.cudnn.is_available.md:18]: 可以的，如果为None或者空串会返回False，其余情况为True

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.MultiHeadAttention.md (hunk 1 above)
Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.MultiHeadAttention.md]: done！

File: docs/guides/model_convert/convert_from_pytorch/api_difference/sparse/torch.sparse.sum.md (hunk 3 above)
Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/sparse/torch.sparse.sum.md]: done！

File: docs/guides/model_convert/convert_from_pytorch/api_difference/sparse/torch.sparse.sum.md (hunk 4 above)
Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/sparse/torch.sparse.sum.md:24]: done！

**Review by zhwesky2010 — APPROVED** [review]
LGTM
